core/shared/src/test/scala/fs2/ParEvalMapSuite.scala

```diff
@@ -66,17 +66,13 @@ class ParEvalMapSuite extends Fs2Suite {
     }
 
     test("may not be preserved in parEvalMapUnordered") {
-      run(_.parEvalMapUnorderedUnbounded(identity)).assertEquals(List(1, 2, 3))
-    }
-
-    def run(pipe: Pipe[IO, IO[Int], Int]) =
       Stream
         .emits(List(3, 2, 1))
         .map(i => IO.sleep(50.millis * i.toLong).as(i))
```

Comment:
Although, based on the test, it looks like a very specific order is expected :)

Instead we should rewrite this to use `executeEmbed` so it won't flake.